api/src/main/kotlin/team/backend/common/logging/LoggingFilter.kt

```diff
@@ -0,0 +1,37 @@
+package team.backend.common.logging
```

Comment:
Shouldn't this class be in the core package, not the api package?

Reply:
It is not clear whether the implementation of WebFilter should be in the core module.

For monolithic types, both core and api are irrelevant, but if more bootJar modules are added, it may be better to add to each xxx-api module.